lib/Cpdf.php

```diff
@@ -6230,15 +6230,21 @@ function addJpegFromFile($img, $x, $y, $w = 0, $h = 0)
         // attempt to add a jpeg image straight from a file, using no GD commands
         // note that this function is unable to operate on a remote file.
 
-        if (!file_exists($img)) {
-            return;
+        if(substr($img, 0, 5) == 'data:') {
+            $filename = 'data-'.hash('md4',$img);
+        }
```

Comment:
Now wondering if we should apply this same logic elsewhere where the filename is used as a caching key.